packages/salesforcedx-vscode-apex/src/commands/apexActionController.ts

```diff
@@ -44,10 +45,14 @@ export class ApexActionController {
           if (!metadata) {
             throw new Error(nls.localize('extraction_failed', type));
```

Comment:
This label name makes me think of going to the dentist ;). A different name related to the operation just performed?

Reply:
lol. Sure